source/rate-limit.ts

```diff
@@ -64,6 +64,7 @@ const promisifyStore = (passedStore: LegacyStore | Store): Store => {
 						totalHits: number,
 						resetTime: Date | undefined,
 					) => {
+						/* istanbul ignore if */
```

Comment:
IMO we don't really need these istanbul ignore comments. I'm OK with not having 100% test coverage; this kind of feels like we're faking it, anyways.